open-metadata-implementation/integration-services/search-integrator/search-integrator-api/src/main/java/org/odpi/openmetadata/integrationservices/search/ffdc/SearchIntegratorErrorCode.java

```diff
@@ -0,0 +1,112 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.integrationservices.search.ffdc;
+
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageDefinition;
+import org.odpi.openmetadata.frameworks.auditlog.messagesets.ExceptionMessageSet;
+
+/**
+ * The SearchIntegratorErrorCode error code is used to define first failure data capture (FFDC) for errors that
+ * occur when working with the Integration Services.  It is used in conjunction with all exceptions,
+ * both Checked and Runtime (unchecked).
+ *
+ * The 5 fields in the enum are:
+ * <ul>
+ *     <li>HTTP Error Code for translating between REST and JAVA - Typically the numbers used are:</li>
+ *     <li><ul>
+ *         <li>500 - internal error</li>
+ *         <li>400 - invalid parameters</li>
+ *         <li>404 - not found</li>
+ *         <li>409 - data conflict errors - eg item already defined</li>
+ *     </ul></li>
+ *     <li>Error Message Id - to uniquely identify the message</li>
+ *     <li>Error Message Text - includes placeholder to allow additional values to be captured</li>
+ *     <li>SystemAction - describes the result of the error</li>
+ *     <li>UserAction - describes how a user should correct the error</li>
+ * </ul>
+ */
+public enum SearchIntegratorErrorCode implements ExceptionMessageSet
+{
+
+    INVALID_CONNECTOR(400,"OMIS-SEARCH-INTEGRATOR-400-001",
+                  "Integration connector {0} is not of the correct type to run in the {1} integration service.  It must inherit from {2}",
+                  "The integration service fails to start and this in turn causes the integration daemon to fail.",
+                  "The connector was configured through the administration calls for the integration service." +
+                              "Either move it to an appropriate integration service or update the connector implementation " +
+                              "to inherit from the correct class."),
+
+    NULL_CONTEXT(400,"OMIS-SEARCH-INTEGRATOR-500-001",
```

Comment:
shouldn't the first parameter be 500 to match the 500 in the message identifier

Reply:
Same pattern is applied throughout the integration connectors, they return 400 for null context